code/modules/hivemind/machines.dm

```diff
@@ -0,0 +1,486 @@
+	max_health = 60
+	evo_points_required = 100 //it's better to wait a bit
+	cooldown_time = 120 SECONDS
+	global_cooldown = TRUE
+	icon_state = "antenna"
+	var/list/appeal = list("They are", "He is", "All of them are", "I'm")
+	var/list/act = list("looking", "already", "coming", "going", "done", "joined", "connected", "transfered")
+	var/list/article = list("for", "with", "to")
+	var/list/pattern = list("us", "you", "them", "mind", "hive", "machine", "help", "hell", "dead", "human", "machine")
+
+
+//this one is slow, careful with it
+/obj/structure/hivemind_machine/babbler/use_ability()
+	flick("[icon_state]-anim", src)
+	var/msg_cycles = rand(1, 2)
+	var/msg = ""
+	for(var/i = 1 to msg_cycles)
+		var/list/msg_words = list()
+		msg_words += pick(appeal)
+		msg_words += pick(act)
+		msg_words += pick(article)
+		msg_words += pick(pattern)
+
+		var/word_num = 0
+		for(var/word in msg_words) //corruption
+			word_num++
+			if(prob(50))
+				var/corruption_type = pick("uppercase", "noise", "jam", "replace")
+				switch(corruption_type)
+					if("uppercase")
+						word = uppertext(word)
+					if("noise")
+						word = pick("z-z-bz-z", "hz-z-z", "zu-zu-we-e", "e-e-ew-e", "bz-ze-ew")
+					if("jam") //word jamming, small Max Headroom's cameo
+						if(lentext(word) > 3)
+							var/entry = rand(2, lentext(word)-2)
+							var/jammed = ""
+							for(var/jam_i = 1 to rand(2, 5))
+								jammed += copytext(word, entry, entry+2) + "-"
+							word = copytext(word, 1, entry) + jammed + copytext(word, entry)
+					if("replace")
+						if(prob(50))
+							word = pick("CORRUPTED", "DESTRUCTED", "SIMULATATED", "SYMBIOSIS", "UTILIZATATED", "REMOVED", "ACQUIRED")
+						else
+							word = pick("REALLY WANT TO", "TAKE ALL OF THAT", "ARE YOU ENJOY IT", "NOT SUPPOSED TO BE", "THERE ARE NO ESCAPE", "HELP US")
+			if(word_num != msg_words.len)
+				word += " "
+			msg += word
+		msg += pick(".", "!")
+		if(i != msg_cycles)
+			msg += " "
+	global_announcer.autosay(msg, "unknown")
+
+
+//SHRIEKER
+//this machine just stuns enemies
+/obj/structure/hivemind_machine/screamer
+	name = "subjugator"
+	desc = "Head in metal carcass. Still alive, still functional, still screaming."
+	max_health = 100
+	icon_state = "head"
+	evo_points_required = 200
+	cooldown_time = 30 SECONDS
+
+
+/obj/structure/hivemind_machine/screamer/activate()
+	for(var/mob/living/target in view(src))
+		if(target.stat == CONSCIOUS && target.faction != "hive")
+			if(isdeaf(target))
+				continue
+			if(istype(target, /mob/living/carbon/human))
+				var/mob/living/carbon/human/H = target
+				if(istype(H.l_ear, /obj/item/clothing/ears/earmuffs) && istype(H.r_ear, /obj/item/clothing/ears/earmuffs))
+					continue
+			use_ability(target)
+	set_cooldown()
+
+
+/obj/structure/hivemind_machine/screamer/use_ability(mob/living/target)
+	target.Weaken(5)
+	target << SPAN_WARNING("You hear a terrible shriek, there are many voices, a male, a female, synthetic noise.")
```

Comment:
I'm pretty it should be "a male, a female and synthetic noise" since you usually end the final clause with an and.